documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -310,106 +103,134 @@ Controller services can be configured in three different ways:
+Segment store supports security configuration via a properties file (`config.properties`) or JVM system properties. The table 
+below lists its TLS and auth parameters and sample values. For a detailed discription of these parameters refer to
+[this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
 
  | Configuration Parameter| Example Value |
  |:-----------------------:|:-------------|
- | `pravegaservice.security.tls.enable` | true |
- | `pravegaservice.security.tls.server.certificate.location` | /etc/secrets/segmentstore01.pem |
- | `pravegaservice.security.tls.server.privateKey.location` | /etc/secrets/segmentstore01.key.pem |
- | `pravegaservice.zk.connect.security.enable` | false <sup>2</sup> |
- | `pravegaservice.zk.connect.security.tls.trustStore.location` | /etc/secrets/client.truststore.jks |
- | `pravegaservice.zk.connect.security.tls.trustStore.pwd.location` | /etc/secrets/client.truststore.jks.password |
- | `autoScale.controller.connect.security.tls.enable` | true |
- | `autoScale.controller.connect.security.tls.truststore.location` | /etc/secrets/segmentstore01.key.pem |
- | `autoScale.controller.connect.security.auth.enable` | true |
- | `autoScale.security.auth.token.signingKey.basis` | a-secret-value <sup>1</sup>|
- | `autoScale.controller.connect.security.tls.validateHostName.enable` | true |
-
-[1]: It is assumed here that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side TLS configuration properties via the `pravegaservice.zk.*` properties.
+ | `pravegaservice.security.tls.enable` | `true` |
+ | `pravegaservice.security.tls.server.certificate.location` | `/etc/secrets/segmentstore01-server-cert.crt` |
+ | `pravegaservice.security.tls.certificate.autoReload.enable` | `false` |
+ | `pravegaservice.security.tls.server.privateKey.location` | `/etc/secrets/segmentstore01-server-key.key` |
+ | `pravegaservice.zk.connect.security.enable` | `false` <sup>2</sup> |
+ | `pravegaservice.zk.connect.security.tls.trustStore.location` | Unspecified <sup>2</sup>|
+ | `pravegaservice.zk.connect.security.tls.trustStore.pwd.location` | Unspecified <sup>2</sup>|
+ | `autoScale.controller.connect.security.tls.enable` | `true` |
+ | `autoScale.controller.connect.security.tls.truststore.location` | `/etc/secrets/ca-cert.crt` |
+ | `autoScale.controller.connect.security.auth.enable` | `true` |
+ | `autoScale.security.auth.token.signingKey.basis` | `a-secret-value` <sup>1</sup>|
+ | `autoScale.controller.connect.security.tls.validateHostName.enable` | `true` |
+ | `pravega.client.auth.loadDynamic` | `false` |
+ | `pravega.client.auth.method` | `Basic` |
+ | `pravega.client.auth.token` | Base64-encoded value of 'username:password' string | 
+
+[1]: It is assumed here that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side TLS 
+configuration properties via the `pravegaservice.zk.*` properties.
```

Comment:
"via the `pravegaservice.zk.*` properties." -> "via these properties."

Reply:
Fixed.